### ISSUE TITLE
How to resume training from a specific epoch ?

### ISSUE THREAD

**Issue by nikhilbyte** [bayer-science-for-a-better-life/contrastive-reconstruction#2]
I was training the model and I stopped training at 13th epoch. When I tried to resume training from the 13th epoch it started training from the 1st epoch. 
P.S. I don't want to finetune the model, I just want to resume the training from the 13th epoch.
Thankyou

**Comment by jonasd4** [comment]
You can specify the path to a pretrained model with the `--pretrained` argument. The weights of this model will then be used as an initalization for training. As we keep the learning rate constant over time, it does not make a difference if you start from the 1st or the 13th epoch. However, the optimizer state is currently not saved during training. Therefore, it might lead to slight different results when stopping and continuing the training. I am working on this and will include exact resuming of the training in a future version.
